content/pt-br/docs/tasks/run-application/force-delete-stateful-set-pod.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Forçar a Exclusão de Pods de um StatefulSet
+content_type: task
+weight: 70
+---
+
+<!-- overview -->
+Esta página mostra como excluir Pods que fazem parte de um
+{{< glossary_tooltip text="StatefulSet" term_id="StatefulSet" >}} e
+explica as considerações que devem ser levadas em conta ao fazer isso.
+
+## {{% heading "prerequisites" %}}
+
+- Esta é uma tarefa relativamente avançada e pode violar algumas das propriedades 
+  inerentes ao StatefulSet.
+- Antes de prosseguir, familiarize-se com as considerações listadas abaixo.
+
+<!-- steps -->
+
+## Considerações sobre StatefulSet
+
+Na operação normal de um StatefulSet, **nunca** há necessidade de forçar a exclusão de um Pod.
+O [controlador de StatefulSet](/docs/concepts/workloads/controllers/statefulset/) é responsável por criar,
+escalar e excluir os membros do StatefulSet. Ele tenta garantir que o número especificado de Pods,
+do ordinal 0 até N-1, esteja ativo e pronto. O StatefulSet garante que, a qualquer momento,
+exista no máximo um Pod com uma determinada identidade em execução no cluster. Isso é chamado de semântica
+*no máximo um* fornecida por um StatefulSet.
+
+A exclusão forçada manual deve ser realizada com cautela, pois tem o potencial de violar a semântica de *no máximo um*
+inerente ao StatefulSet. StatefulSets podem ser usados para executar aplicações distribuídas e em cluster que
+necessitam de uma identidade de rede estável e armazenamento estável. Essas aplicações frequentemente possuem
+configurações que dependem de um conjunto fixo de membros com identidades fixas. Ter múltiplos membros com a mesma
+identidade pode ser desastroso e pode levar à perda de dados (por exemplo, cenário de split brain em sistemas baseados em quórum).
```

Comment:
Já que não temos uma boa tradução para o termo, o que tu achas de grafar em itálico neste caso?
```suggestion
identidade pode ser desastroso e pode levar à perda de dados (por exemplo, cenário de _split brain_ em sistemas baseados em quórum).
```